app-tests/gromacs/gromacs_simple.py

```diff
@@ -0,0 +1,129 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
+
+@rfm.simple_test
+class simple_gromacs_check(gromacs_check):
+    maintainers = ['mszpindler']
+    use_multithreading = False
+    executable_opts += ['-dlb yes', '-ntomp 1', '-npme -1']
+    pe_release = '21.12'
+    gromacs_version = '2021.5'
+    num_nodes = 1
+
+    benchmark_info = parameter([
+        ('benchMEM', -1.08735e+06, 0.001),
+        ('benchPEP', -1.08735e+06, 0.001), # Fix it
+    ], fmt=lambda x: x[0], loggable=True)
+
+    allref = {
+	1: {
+            'cascadelake': {  # Copied from Cascade Lake results 
+                'benchMEM': (92.043, None, None, 'ns/day'),
+                'benchPEP': (0.561, None, None, 'ns/day'), 
+            }
+        }
+    }
+
+    @run_after('init')
+    def setup_modules(self):
+        if self.current_system.name in ('lumi'):
+            if self.nb_impl == 'cpu':
+                self.modules = ['GROMACS-{gromacs_version}-cpeGNU-{pe_release}-CPU']
```

Comment:
Do we need to specify here the full name of the module? The other tests are written for the default version of the module. Why this one needs the full name?